### PR TITLE
139 notifications add two versions of each email for admins and users

### PR DESCRIPTION
**Pull request by marcinkrzeminski**

### CODE REVIEW

**Review by dawidurbanski — CHANGES_REQUESTED** [review]
This works and PR looks great. Just one thing. It only added sending two emails (one for admin and one for customer), to the "appointment created" action.

We want the same for all the other actions.

- [x] Appointment created
    - [x] admin
    - [x] user
- [ ] Appointment updated
    - [ ] admin
    - [x] user
- [ ] Appointment cancelled
    - [ ] admin
    - [x] user
- [ ] Appointment no show
    - [ ] admin
    - [x] user
- [ ] Appointment confirmed
    - [ ] admin
    - [x] user

**Comment by marcinkrzeminski** [comment]
@dawidurbanski please check again. Admin notifications covered for all types of emails.

File: assets/backend/admin/components/AppointmentsTableFull/AppointmentsTableFull.tsx [hunk 1]
```diff
@@ -193,7 +193,7 @@ function TableRow({
 						[styles[row.status]]: true,
 					})}
 				>
-					{row.status}
+					{row.status.replace('_', ' ')}
```
Comment (dawidurbanski) [inline assets/backend/admin/components/AppointmentsTableFull/AppointmentsTableFull.tsx:196]: nie ma potrzeby robić tego replace tak?

**Review by dawidurbanski — APPROVED** [review]
LGTM

**Pull request merged**